README.md

```diff
@@ -40,25 +40,40 @@ Install gems for site:
 bundle install
 ```
 
-Run server:
+Install Netlify CLI:
+
+```bash
+brew install netlify-cli
+```
+
+### Run server
+
+Run server using Netlify Dev:
 
 ```bash
-bundle exec jekyll serve
+netlify dev
 ```
 
 With the server running you can access the site on
-[http://localhost:4000](http://localhost:4000).
+[http://localhost:8888](http://localhost:8888).
 
-## Additional Notes
+## Additional notes
 
 ### Verify pages render correctly on phone
 
 You can run the server on your computer and then verify the pages render
-correctly on a phone.  By default the server only binds to localhost, so it is
-not accessible on your local network.  Add `--host 0.0.0.0` to the `bundle exec`
-command to have the server bind to the external IP addresses on your computer.
+correctly on a phone. There are two ways to do that:
+
+* Access the IP address of your computer on your local network from your phone.
+
+* Run `netlify dev --live` to create a publically accessible tunnel that can
```

Comment:
I got this error when runing --live
```
◈ Error: no siteId defined, did you forget to run netlify init or netlify link?
```

Should we use some sort of shared site or create our own account?

Reply:
It was pretty straightforward to just create my own so I don't think we need to document it here perse...just mention that you'll need to create and login or `netlify link` your site/account.

Reply:
Can you try linking it to the existing site? Or so you have to have a Netlify login of your own?